.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can we get rid of the empty file?

Reply:
As I told @ptrgags this is not just an empty file. This file is explicitly declaring that we are choosing to use the default prettier settings. Without this editors, extensions and even the cli without the `--no-config` option have a chance of picking up a different configuration.

For example, my default VSCode settings prefer single quote over double quote. Without this file or setting it in `.vscode/settings.json` my prettier extension changes every quote when I save and I'd have to manually change it before I commit.

From prettier's [docs](https://prettier.io/docs/en/configuration.html), note that it doesn't specify any sort of "project boundary" so someone could have a config at the root that affects all projects:

> The configuration file will be resolved starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn’t) found.

From the prettier extension's [docs](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

> There are multiple options for configuring Prettier with this extension. You can use [VS Code settings](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode#prettier-settings), [prettier configuration files](https://prettier.io/docs/en/configuration.html), or an `.editorconfig` file. The VS Code settings are meant to be used as a fallback and are generally intended only for use on non-project files. **It is recommended that you always include a prettier configuration file in your project specifying all settings for your project.** This will ensure that no matter how you run prettier - from this extension, from the CLI, or from another IDE with Prettier, the same settings will get applied.